gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
 kotlin = "1.7.22"
-okio = "3.2.0"
+okio = "3.3.0"
```

Comment:
hm, seems `okio 3.3.0` is only for kotlin >= 1.8
so it should be merged after https://github.com/saveourtool/save-cli/pull/470

Reply:
Let's may be update `okio` after #470 in separate PR, and here update all others? @nulls @0x6675636b796f75676974687562

Reply:
agree

Reply:
@kgevorkyan #481